azafea/event_processors/endless/ping/v1/migrations/9214d7c1d7d9_alpha_2_ping.py

```diff
@@ -0,0 +1,29 @@
+# type: ignore
+
+"""Allow only alpha2 country codes in ping_v1 table
+
+Revision ID: 9214d7c1d7d9
+Revises: f2d6141dbece
+Create Date: 2020-05-14 17:24:17.029109
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '9214d7c1d7d9'
+down_revision = 'f2d6141dbece'
+branch_labels = None
+depends_on = None
```

Comment:
Is this right? Doesn't it depend on the previous migration that changed the constraint?

Reply:
I’ve merged master in this branch on my computer, no new deployment on ping and activation has been added since. I can check twice, just to be sure.

Reply:
It’s OK for both.

Reply:
Maybe I don't quite understand how alembic works. I've read the documentation and now I think I understand:

- The ordering dependency is expressed by `down_revision = 'f2d6141dbece'`
- `depends_on` allows additional edges to be add to the dependency graph